src/translations/screens/Tickets.ts

```diff
@@ -119,6 +119,14 @@ const TicketsTexts = {
       ),
       transportModes: _('Flere reisemåter', 'Several travel modes'),
     },
+    carnet: {
```

Comment:
Is this used? We are not selling carnets in the app.

Reply:
But we are showing them if sold on the webshop?

Reply:
We are not selling but we are showing and hence we are using transport modes. The rest of them are not being used actively right now, But since I have modified the ticket component and we are resolving the title of tickets dynamically using `ticket type` and carnet is a type of ticket, typescript gives an error. So had to add values for carnet as well. 
`t(TicketsTexts.availableTickets[ticketType].title);`
Thought it should be no harm as carnet should be added in app eventually. But the other option is to exclude carnet from ticket component and remove here as well.